Voycar.Api.Web/Features/Members/Endpoints/Put/Personal/Endpoint.cs

```diff
@@ -0,0 +1,66 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Put.Personal;
+
+using Entities;
+using FluentValidation.Results;
+using Org.BouncyCastle.Crypto.Digests;
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok, BadRequest<string>>, Mapper>
```

Comment:
```suggestion
public class Endpoint : Endpoint<Request, Ok, Mapper>
```
`BadRequest` haben wir glaube ich bisher generell nicht als Response angegeben, muss auch nicht, kann Fast Endpoints so. Und spezifisch`BadRequest<string>` stimmt auch nicht mehr.